pkg/api/config/config.go

```diff
@@ -64,6 +64,7 @@ type OpenIDConfig struct {
 }
 
 type OpenIDProviderConfig struct {
+	Name         string
```

Comment:
Where is this set? Can we have an example configuration file added to the PR?
For example an update of https://github.com/project-zot/zot/blob/main/examples/config-openid.json

Reply:
Done, I've added an example in [config-openid.json](https://github.com/project-zot/zot/blob/main/examples/config-openid.json)

Reply:
and yes @rchincha  it's related to https://github.com/project-zot/zui/pull/375 :)